commands/create-registry.ts

```diff
@@ -0,0 +1,127 @@
+
```

Comment:
This file should not exist/ be registered in this commit, you should maintain functionality only from delete registry.